packages/squiggle-lang/src/rescript/Distributions/DistributionOperation.resi

```diff
@@ -14,6 +14,7 @@ type outputType =
   | Dist(genericDist)
   | Float(float)
   | String(string)
+  | FloatArray(array<float>)
```

Comment:
Should this be more general? Like Array(outputType)?

Reply:
I think I want to press back against this, until we get a few more types here. Changing it to something more general would mean needing to handle error cases and similar, even though this will never produce error cases.